zeebe/clients/zeebe-client-spring/src/main/java/io/camunda/zeebe/spring/client/configuration/AnnotationProcessorConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.spring.client.configuration;
+
+import io.camunda.zeebe.spring.client.annotation.customizer.ZeebeWorkerValueCustomizer;
+import io.camunda.zeebe.spring.client.annotation.processor.AbstractZeebeAnnotationProcessor;
+import io.camunda.zeebe.spring.client.annotation.processor.ZeebeAnnotationProcessorRegistry;
+import io.camunda.zeebe.spring.client.annotation.processor.ZeebeDeploymentAnnotationProcessor;
+import io.camunda.zeebe.spring.client.annotation.processor.ZeebeWorkerAnnotationProcessor;
+import io.camunda.zeebe.spring.client.event.ZeebeClientEventListener;
+import io.camunda.zeebe.spring.client.jobhandling.JobWorkerManager;
+import java.util.List;
+import org.springframework.context.annotation.Bean;
+
+public class AnnotationProcessorConfiguration {
```

Comment:
💭  I generally also think this better fits into the sdk starter project => +1 to move everything there